controllers/image_generator.go

```diff
@@ -835,8 +836,8 @@ func (r *ImageGenerator) updateImageConfigMap() error {
 		igLogger.Info("Setting FIPS mode")
 	}
 
-	// set to default image
-	if cm.Data["PODVM_IMAGE_URI"] == "" && os.Getenv("RELATED_IMAGE_PODVM_OCI") != "" {
+	// set to default image which is currently not built for s390x
+	if cm.Data["PODVM_IMAGE_URI"] == "" && os.Getenv("RELATED_IMAGE_PODVM_OCI") != "" && runtime.GOARCH != "s390x" {
```

Comment:
Do we need this conditional? My worry is with this change there won't be any way to test pre-built image for s390x without code change.
Can't it be ensured that RELATED_IMAGE_PODVM_OCI is empty for s390x?